### ISSUE TITLE
mutect2 homozygous alt GT annotation

### ISSUE THREAD

**Issue by jkobject** [broadinstitute/gatk#7650]
## Feature request

Running mutect2 (GATK v4.2.2) in tumor only mode on WES samples and asking it to use/keep germline and PON mutations.
I have noticed that mutect never sets the GT field to be 1/1 (homozygous alt). even with 0 ref and 300 alt reads it will say 0/1, is there a way to set a threshold to define a point on which not to call heterozygosity?

Thanks!

**Comment by your-highness** [comment]
In a recent tumor-only analysis I also observed this. I could not find any documentation why no hom_alt will be reported. There is some dicussion in [Chapter II E in "Notes on MUTECT2"](https://raw.githubusercontent.com/broadinstitute/gatk/master/docs/mutect/mutect.pdf) but it is still puzzling to me. I assume it is related to the fact that homozygous calls _- as opposed to heterozygous calls -_ do not follow a "traceable" 50% ratio and could therefore also be of somatic origin.

**Comment by jkobject** [comment]
Yes I read this document but it still doesn't make sense to me either..

**Comment by davidbenjamin** [comment]
@jkobject Discrete genotypes are not meaningful in somatic calling and Mutect2 has long simply emitted 0/1 for all variants as a placeholder.  We're probably going to get rid of the GT entirely soon.
